hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3001,6 +3010,43 @@
       stage: V2
       tests: []
     }
+    {
```

Comment:
@cindychip 
@weicaiyang 
@msfschaffner 
@moidx 

i struggled a bit with these. There is A LOT of overlap around our life cycle test cases. I wonder if it makes sense to consolidate a bit.

There are multiple tests that check rv jtag is inaccessible (tap_strap_sampling and the lc tests).
I'm not sure if the answer is we should combine these, or simply make sequences re-usable between them

Reply:
LGTM
I think we allow to have some overlap. And this entry has some new test points. Perhaps we can let test developer to decide update the existing test or add a  new one.